compose/compose.go

```diff
@@ -158,10 +165,17 @@ func getOffset(idx int) int {
 	return idx
 }
 
-func initService(basename string, idx, grpcPort int) service {
+func initService(basename string, idx, grpcPort int, aliasName string) service {
 	var svc service
 
-	svc.name = name(basename, idx)
+	cont_name := func() string {
```

Comment:
This is a simple if-else, you don't need a function for this. Also, don't use snake case.